src/compiler/compile.ts

```diff
@@ -69,6 +69,19 @@ export function compile(
         trackInfos = getRelativeTrackInfo(specCopy, theme).trackInfos;
     }
 
+    // publish the fixed spec
+    // used for alt-gosling
+    try {
+        if (PubSub) {
+            PubSub.publish('specTraversed', {
+                id: specCopy.id,
+                data: specCopy
+            });
+        }
+    } catch (e) {
+        //
+    }
+
```

Comment:
Just noticed this `try` /`catch` without any error handling. Is it expected that this could fail with some frequency and we wouldn't want to propagate that to developer?

Reply:
I had already taken it out to mimic https://github.com/gosling-lang/gosling.js/blob/71a2a28b0739f452389990603c7bff7911536ffd/src/tracks/gosling-track/gosling-track.ts#L478-L481 (see latest changes)

I had originally used this similar to the deprecated rawData publishing https://github.com/gosling-lang/gosling.js/blob/f17e69f8ea9d83e9c574e864b7dfca94a49fa328/src/tracks/gosling-track/gosling-track.ts#L921

Reply:
Ah got it! Sorry I missed the `outdated` tag